README.md

```diff
@@ -1,33 +1,83 @@
 
-EasyPost is a simple shipping API. You can sign up for an account at https://easypost.com
+EasyPost, the simple shipping solution. You can sign up for an account at https://easypost.com.
 
-## Documentation
-
-Up-to-date documentation at: https://www.easypost.com/docs/api/csharp
-
-## Installation
+## Install
 
-The easiest way to add EasyPost to your project is with the NuGet package manager.
-
-```
+```powershell
 Install-Package EasyPost-Official
 ```
 
-See NuGet docs for instructions on installing via the [dialog](http://docs.nuget.org/docs/start-here/managing-nuget-packages-using-the-dialog) or the [console](http://docs.nuget.org/docs/start-here/using-the-package-manager-console).
+See NuGet docs for additional instructions on installing via the [dialog](http://docs.nuget.org/docs/start-here/managing-nuget-packages-using-the-dialog) or the [console](http://docs.nuget.org/docs/start-here/using-the-package-manager-console).
 
 ## Usage
 
-The EasyPost API consists of many object types. There are several attributes that are consistent across all objects:
+A simple create & buy shipment example:
 
-* `id` -- Guaranteed unique identifier of the object.
-* `created_at`/`updated_at`  -- Timestamps of creation and last update time.
+```csharp
+using System;
+using System.Collections.Generic;
+using EasyPost;
+using Newtonsoft.Json;
+
+namespace shipments
+{
+    class createShipment
+    {
+        static void Main()
+        {
+            EasyPost.ClientManager.SetCurrent(Environment.GetEnvironmentVariable("EASYPOST_API_KEY"));
+
+            Dictionary<string, object> fromAddress = new Dictionary<string, object>() {
+                { "name", "Dr. Steve Brule" },
+                { "street1", "417 Montgomery Street" },
+                { "street2", "5th Floor" },
+                { "city", "San Francisco" },
+                { "state", "CA" },
+                { "country", "US" },
+                { "zip", "94104" },
+                { "phone", "4153334444" }
+            };
+
+            Dictionary<string, object> toAddress = new Dictionary<string, object>() {
+                { "company", "EasyPost" },
+                { "street1", "417 Montgomery Street" },
+                { "street2", "Floor 5" },
+                { "city", "San Francisco" },
+                { "state", "CA" },
+                { "country", "US" },
+                { "zip", "94104" },
+                { "phone", "415-379-7678" }
+            };
+
+            Dictionary<string, object> parcel = new Dictionary<string, object>() {
+                { "length", 8 },
+                { "width", 6 },
+                { "height", 5 },
+                { "weight", 10 },
+            }
+
+            Shipment shipment = Shipment.Create(new Dictionary<string, object>() {
+                { "from_address", fromAddress },
+                { "to_address", toAddress },
+                { "parcel", parcel },
+            });
+
+            shipment.Buy(shipment.LowestRate());
```

Comment:
This needs to be async

Reply:
Perhaps I pull it out of a class then and only have it be a barebones example, or do you mean the calls to the functions need to be async? Either way, care to provide suggestions on the review of what's needed where?

Reply:
I believe I've added async/await where necessary. Ready for a double check.